js/controller.js

```diff
@@ -1,13 +1,5 @@
 angular.module('Come2HelpController', ['ngResource']);
```

Comment:
Yes, we should improve the name of this module, but I think is independent of this pull request.